### ISSUE TITLE
Bookmarks from Booklice not showing up

### ISSUE THREAD

**Issue by michielbdejong** [solid-contrib/data-modules#78]
See https://pdsinterop.org/conventions/bookmark/#booklice

![Screenshot 2024-01-22 at 16 26 52](https://github.com/solid-contrib/data-modules/assets/408412/fe5e4ad9-4e2b-4d18-a649-fd8806f0a1a5)

**Comment by soltanireza65** [comment]
Ah,
it uses completely different predicates

```turtule
<#1702557925503> a <https://www.w3.org/ns/activitystreams#Note>;
    <https://www.w3.org/ns/activitystreams#name> "bravo";
    <https://www.w3.org/ns/activitystreams#content> "";
    <https://www.w3.org/ns/activitystreams#url> <https://www.google.com>;
    <https://www.w3.org/ns/activitystreams#actor> <https://rosano.solidcommunity.net/profile/card#me>;
    <https://www.w3.org/ns/activitystreams#published> "2023-12-14T12:45:25.503Z".
```

**Comment by michielbdejong** [comment]
Yep :) Good opportunity to show the value of our data module!

**Comment by michielbdejong** [comment]
@soltanireza65 I'll take this one

**Comment by michielbdejong** [comment]
```
<#1705937062715> a <https://www.w3.org/ns/activitystreams#Note>;
    <https://www.w3.org/ns/activitystreams#name> "my title";
    <https://www.w3.org/ns/activitystreams#content> "my note";
    <https://www.w3.org/ns/activitystreams#url> <https://pondersource.com>;
    <https://www.w3.org/ns/activitystreams#actor> <https://michielbdejong.solidcommunity.net/profile/card#me>;
    <https://www.w3.org/ns/activitystreams#published> "2024-01-22T15:24:22.715Z";
    <https://www.w3.org/ns/activitystreams#tag> "#very-important".
